src/bun.js/bindings/exports.zig

```diff
@@ -1704,6 +1703,13 @@ pub const ZigConsoleClient = struct {
                     value: JSValue,
                     is_symbol: bool,
                 ) callconv(.C) void {
+Output.prettyErrorln("KEY: {s} {d} {d}\nVALUE: {d} {d}", .{
```

Comment:
don't forget to not commit this

Reply:
Purged in https://github.com/oven-sh/bun/pull/1674/commits/76f64875695528d6a5c83ebb88465cdca14e02b0